packages/@ember/application/tests/application_instance_test.js

```diff
@@ -165,7 +171,7 @@ moduleFor(
     }
 
     ['@test can build and boot a registered engine'](assert) {
-      assert.expect(11);
+      assert.expect(12);
```

Comment:
Where does the extra assertion come from?

Reply:
I'm going to see if I can make it clear, but it requires some messing with QUnit.